AutonomousDemo/Velocity.cs

```diff
@@ -23,7 +23,6 @@ public Velocity(double speed = 0, double direction = Math.PI/2.0)
 
         public static Velocity operator +(Velocity vIntial, Velocity vFinal)
         {
-
             var totalSpeed = vIntial.Speed + vFinal.Speed;
             var totalDirection = vIntial.Direction + vFinal.Direction;
```

Comment:
This is not an accurate representation of what happens when you add two vectors. Consider: 20m/s north + 20m/s north does not equal 40 m/s west.